cardinal/server/handler/tx.go

```diff
@@ -30,22 +30,15 @@ type Transaction = sign.Transaction
 
 // PostTransaction godoc
 //
-//	@Summary		Submit a transaction to Cardinal
-//	@Description	Submit a transaction to Cardinal / Create a Persona transaction to Cardinal
-//	@Accept			application/json
-//	@Produce		application/json
-//	@Success		200	{object}	PostTransactionResponse
-//	@Failure		400	{string}	string	"Invalid transaction request"
-//
-// PostTransaction with label godoc
-//
-//	@Router			/tx/game/{txType} [post]
-//	@Param			txType	path	string	true	"label of the transaction that wants to be submitted"
-//
-// PostTransaction with Persona godoc
-//
-//	@Router			/tx/persona/create-persona [post]
-//	@Param			txBody	body	Transaction	true	"Transaction details"
+//	@Summary      Submit a message to Cardinal
```

Comment:
A transaction is message + signature, nonce, namespace, etc. Technically, this endpoint takes in a transaction that contains a message. How can we better phrase this?